flink-core/src/test/java/org/apache/flink/api/common/io/GlobFilePathFilterTest.java

```diff
@@ -138,4 +141,30 @@ public void doubleStarPattern() {
 		assertFalse(matcher.filterPath(new Path("a/b")));
 		assertFalse(matcher.filterPath(new Path("a/b/c")));
 	}
+
+	@Test(expected = NullPointerException.class)
+	public void testIncluePatternIsNull() {
+		new GlobFilePathFilter(
+			null,
+			Collections.<String>emptyList());
+	}
+
+	@Test(expected = NullPointerException.class)
+	public void testExcludePatternIsNull() {
+		new GlobFilePathFilter(
+			Collections.singletonList("**"),
+			null);
+	}
+
+	@Test
+	public void testGlobFilterSerializable() throws IOException {
```

Comment:
How about using `CommonTestUtils.createCopySerializable()` here? That you can validate also that the serialized copy behaves valid.